docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,654 @@
+---
+title: Integrating a React Native view into existing native apps
+sidebar_title: Integrated approach
+description: Add Expo and React Native to existing native apps using the "integrated" approach.
+hideTOC: true
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import { PlatformTag } from '~/ui/components/Tag/PlatformTag';
+
+React Native offers a flexible development strategy. Whether you're launching a new app or looking to integrate an existing native app, it provides a powerful solution. You can leverage its capabilities to introduce new features or views incrementally, allowing your team to adopt modern, cross-platform development without a complete rewrite.
+
+This guide will walk you through the steps to add a React Native view into an existing native app using the "integrated" approach.
```

Comment:
```suggestion
This guide will walk you through the steps to add a React Native view into an existing native app. The approach covered here is what we call the "integrated" approach, because React Native and Expo are integrated in the same way that you would any other library.

Another popular technique is what we call the "isolated" approach, where your Expo app is packaged as a library and treated as a black box by the main existing application. A guide for the isolated approach will be available shortly. Now, back to using the "integrated" approach in your existing native app.
```